docs/api/cpp/dynamic_property.rst.in

```diff
@@ -0,0 +1,58 @@
+..
+  SPDX-License-Identifier: CC-BY-4.0
+  Copyright Contributors to the OpenColorIO Project.
+
+DynamicProperty
+===============
+
+.. doxygenclass:: @OCIO_NAMESPACE@::DynamicProperty
+   :members:
+   :undoc-members:
+
+.. doxygentypedef:: @OCIO_NAMESPACE@::ConstDynamicPropertyRcPtr
+.. doxygentypedef:: @OCIO_NAMESPACE@::DynamicPropertyRcPtr
+
+DynamicPropertyValue
+********************
```

Comment:
That should be "=========="; otherwise, it creates a sub-section. The comment also applies to the following sub-sections.